keyboards/gskt00/gskt00.h

```diff
@@ -0,0 +1,57 @@
+#ifndef GSKT00_H
+#define GSKT00_H
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
+    K36, K37, K46, K47, K56, K57, K66, K67, K76, K77, K06, K07, K17, K26, K27, \
+    K34, K35, K44, K45, K54, K55, K64, K65, K75, K05, K15, K16, K25, K24, \
+    K32, K33, K43, K52, K53, K63, K73, K74, K03, K04, K13, K14, K23, \
+    K31, K41, K42, K51, K61, K62, K71, K72, K01, K02, K11, K12, K21, K22, \
+    K30, K40, K50,           K60,                          K00, K10, K20  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07 }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17 }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27 }, \
+    { K30, K31, K32, K33, K34, K35, K36, K37 }, \
+    { K40, K41, K42, K43, K44, K45, K46, K47 }, \
+    { K50, K51, K52, K53, K54, K55, K56, K57 }, \
+    { K60, K61, K62, K63, K64, K65, K66, K67 }, \
+    { KC_NO, K71, K72, K73, K74, K75, K76, K77 } \
+}
+
+#define LAYOUT_60_iso_tsangan( \
+    K36, K37, K46, K47, K56, K57, K66, K67, K76, K77, K06, K07, K17,      K27, \
+    K34, K35, K44, K45, K54, K55, K64, K65, K75, K05, K15, K16, K25, K24, \
```

Comment:
`K24` needs to be moved down to between `K14` and `K23` for `LAYOUT_60_iso_tsangan` to work properly.

Reply:
Changed